app/models/manageiq/providers/ibm_cloud/inventory/parser/power_virtual_servers.rb

```diff
@@ -246,6 +247,19 @@ def networks
     end
   end
 
+  def server_groups
+    byebug
+    collector.server_groups.placement_groups.each do |sgrp|
+      persister.server_groups.build(
+        :type        => "ManageIQ::Providers::IbmCloud::PowerVirtualServers::CloudManager::ServerGroup",
+        :name        => sgrp.name,
+        :policy      => sgrp.policy,
+        :ems_ref     => sgrp.id,
+        :members     => sgrp.members.join(",")
```

Comment:
This is failing for you because there is no `members` column on the `server_groups` table, you probably want to set `server_group => persister.server_groups.lazy_find(server_group_ref)` when parsing vms